src/components/App/App.jsx

```diff
@@ -0,0 +1,18 @@
+import './App.scss'
+import React, { useState } from 'react';
+import Sun from './Assets/sun.svg'
+import Moon from './Assets/half-moon.svg'
+
+function App() {
+    const [isMornig, setIsMorning] = useState(true)
+    const text = isMornig ? 'Good Morgin' : 'Good Night'
+    return (
+        <div className={`wrap wrap__${isMornig ? 'light' : 'dark'}`} >
```

Comment:
don't use `wrap` naming.
using it everywhere will make your code one big mess.
Use meaningful names like `app--layout` or `app--theme`